tests/transforms/test_load_image.py

```
@@ -498,5 +499,70 @@ def test_correct(self, input_param, expected_shape, track_meta):
+        # then verify LoadImage still instantiates and logs warnings.
+        from monai.utils import OptionalImportError
+
+        # Patch SUPPORTED_READERS entries to raise OptionalImportError
+        # This simulates optional packages not being installed
+        from monai.transforms.io.array import SUPPORTED_READERS
+
+        # Patch a few readers to fail (e.g., ITKReader)
+        try:
+            original_itk = SUPPORTED_READERS.get("itk")
+
+            def failing_reader(*args, **kwargs):
```

Ruff (0.15.4)

[warning] Unused function argument: `args`

(ARG001)